developer-experience/readme.md

```diff
@@ -0,0 +1,130 @@
+# Developer Experience
+
+Developer experience refers to the how easy or difficult it is for a developer to perform essential tasks needed to implement a change. A positive developer experience would mean these tasks are relatively easy for the team (see measures below).
+
+The essential tasks are identified below.
+
+Build - Verify that changes are free of syntax error and compile.
+Test - Verify that all automated tests pass.
+Start - Launch end-to-end to simulate execution in a deployed environment.
+Debug - Attach debugger to started solution, set breakpoints, step through code, and inspect variables.
+
+If effort is invested to make these activities as easy as possible, the returns on that effort will be multiply the longer the project runs and the larger the team.
+
+## Goals
+
+- Maximize the amount of time engineers spend on writing code that fulfills story acceptance and done-done criteria.
+- Minimize the amount of time spent manual setup and configuration of tooling
+- Minimize regressions and new defects by making end-to-end testing easy
+
+## Impact
+
+Developer experience can have a significant impact on the efficiency of the day to day execution of the team. A positive experience can pay dividends throughout the lifetime of the project; especially as new developers join the team.

```

Comment:
Dev productivity and reduced friction during the actual CSE engagement period (while very important) are arguably the least interesting aspect... its the accrual of those benefits long after we disengage from active work with the customer which is our big goal, I think. We should call that out here, explicitly.

Reply:
Adding bolded text that calls this out
**Most importantly, the customer will continue to accrue these benefits long after the code-with engagement.**